File: promedio.cpp
#include <iostream>
using namespace std;

int main() {
    float nota1, nota2, nota3, promedio;

    cout << "Ingrese la primera nota: ";
    cin >> nota1;

    cout << "Ingrese la segunda nota: ";
    cin >> nota2;

    cout << "Ingrese la tercera nota: ";
    cin >> nota3;

    promedio = (nota1 + nota2 + nota3) / 3;

    cout << "El promedio es: " << promedio << endl;

    if (promedio >= 3.0) {
        cout << "El estudiante aprueba." << endl;
    } else {
        cout << "El estudiante reprueba." << endl;
    }

    return 0;
}

File: Promedio.kt
fun main() {
    print("Ingrese la primera nota: ")
    val nota1 = readLine()!!.toDouble()

    print("Ingrese la segunda nota: ")
    val nota2 = readLine()!!.toDouble()

    print("Ingrese la tercera nota: ")
    val nota3 = readLine()!!.toDouble()

    val promedio = (nota1 + nota2 + nota3) / 3

    println("El promedio es: $promedio")

    if (promedio >= 3.0) {
        println("El estudiante aprueba.")
    } else {
        println("El estudiante reprueba.")
    }
}
